galsim/base.py

```diff
@@ -1053,27 +1188,29 @@ def drawImage(self, image=None, nx=None, ny=None, bounds=None, scale=None, wcs=N
                         it could be useful if you want to view the surface brightness profile of an
                         object directly, without including the pixel integration.
 
+        To convert from flux (photons/cm^2/s) into image values of analog-to-digital units (ADU), we
+        use optional keywords `area` indicating the telescope collecting area in cm^2, `exptime`
+        indicating the exposure time in s, and `gain` which converts between photons and ADU.
```

Comment:
I'm happy to make this change.  One question though, which @rmandelb brought up, is what to do with the `image.added_flux` attribute.  I *think* it makes sense to keep it in the same units as `gsobj.flux`, which means it doesn't scale with `exptime` or `area` (or `gain` for that matter).  But this does mean that image.added_flux won't be close to image.array.sum().